src/api/java/com/minecolonies/api/util/ItemStackUtils.java

```diff
@@ -521,11 +520,11 @@ private static int getFishingRodLevel(final ItemStack itemStack)
             return 5;
         }
         final int rodDurability = itemStack.getMaxDamage();
-        if (rodDurability <= (Tiers.WOOD.getUses() + MinecoloniesAPIProxy.getInstance().getConfig().getServer().fishingRodDurabilityAdjustT1.get()))
+        if (rodDurability <= (Tiers.WOOD.getUses() + 22))
```

Comment:
What even are these numbers for to begin with (with or without config)?

Reply:
It's to allow for mods that introduce new tiers of fishing rod, so you can tweak which versions require level 2 or 3 huts (e.g. an iron fishing rod might have more durability and require level 2).

So this seems like a bad idea to remove the config setting, as different modpacks might need different values.